bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ClientLibrariesImpl.java

```diff
@@ -203,20 +216,43 @@ private String getHtmlWithInjectedAttributes(String html) {
         return StringUtils.replace(updatedHtml,"<link ", "<link " + cssAttributes.toString());
     }
 
+    /**
+     * Gets the fragment for an attribute, conditioned for include
+     *
+     * @param name - the name of the attribute
+     * @param include - should the attribute be included
```

Comment:
I'd phrase it as: include - {true} to include, {false} otherwise

Reply:
Done in 000c479